content/reference/storage/reading-key-value-pairs.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Reading key-value pairs'
+weight: 80
+---
+
+To get the value for a given key, you can call the `get` method on any
+namespace you've bound to your script:
+
+`NAMESPACE.get(key, [type])`
```

Comment:
This is an ongoing question in the docs, but I wonder if there's the potential for people to interpret `[type]` as a variable type wrapped in an array. I'm guessing that's not the actual syntax, and it's indicating an optional value, right?

Reply:
yes, that's correct. elsewhere we said we use typescript syntax (but I don't know typescript), and it looks like from a cursory glance this isn't the typescript syntax either. i'll change them.

Reply:
actually, this is a good question; we don't want to imply that you can only use this stuff with typescript. any ideas about good wording?

Reply:
I think until we make a team decision on it, maybe it would be best to just show both forms separately? This example can just have the key, and then a little further down, a second example that's `NAMESPACE.get(key, type)` and then the list of potential types. What do you think?

Reply:
pushed a new commit with the change, i like it a lot 👍